src/core/services/layout/layout.js

```diff
@@ -143,6 +163,21 @@
 
   }
 
+
+  /**
+    * Detect if any of the HTML tags has a [md-layouts-disabled] attribute;
+    * If yes, then immediately disable all layout API features
+    *
+    * Note: this attribute should be specified on either the HTML or BODY tags
+    */
+   /**
+    * @ngInject
+    */
+   function detectDisabledLayouts() {
+    var items = document.querySelectorAll('[md-layouts-disabled]');
```

Comment:
I think we should only check `<body>` or `<html>` instead of attribute selectors all over